### PR TITLE
Week four comment translation

### PR DESCRIPTION
**Pull request by panerorenn9541**
Implement the translation feature from week four

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -48,6 +48,15 @@ <h2 style="text-align: center;">
...
           ><option value="5">5 </option></select>
 
+        <p>In which language would you like to see the comments?</p>
+        <select id="commentLanguage" onchange="loadComments()">
+          <option value="en">English </option
+          ><option value="es">Spanish </option
+          ><option value="ca">Catalan </option
+          ><option value="zh">Chinese </option
+          ><option value="pt">Portugese </option
+          ><option value="nl">Dutch </option></select>
```
Comment (huzcn) [inline portfolio/src/main/webapp/index.html]: The style here is weird. Could you fix this and lines in 45-49?

**Review by huzcn — COMMENTED** [review]
Nice!

**Comment by panerorenn9541** [comment]
Fix the formatting of the two select sections

**Review by v01dstar — APPROVED** [review]
LGTM

**Pull request merged**